src/main/scala/scoverage/ScoverageSbtPlugin.scala

```diff
@@ -78,6 +78,8 @@ object ScoverageSbtPlugin extends AutoPlugin {
         Seq(
           Some(s"-Xplugin:${pluginPath.getAbsolutePath}"),
           Some(s"-P:scoverage:dataDir:${crossTarget.value.getAbsolutePath}/scoverage-data"),
+          // This if-else is necessary for compatibility, can be removed in the future
+          if(DefaultScoverageVersion > "1.4.0") Some(s"-P:scoverage:reportTestName") else None,
```

Comment:
Seeing that they default is now `1.4.3` I'd say this check probably doesn't make a ton of sense. Are you able to rebase and update this and just remove the check? We should be safe to include this here now.

Reply:
I removed the check and rebased.